build.sbt

```diff
@@ -0,0 +1,78 @@
+ThisBuild / organization := "za.co.absa"
+ThisBuild / name         := "hermes"
+ThisBuild / version      := "0.1.0-SNAPSHOT"
+ThisBuild / scalaVersion := "2.11.12"
+
+val baseDependencies = List(
+  "com.github.scopt" %% "scopt"     % "4.0.0-RC2",
+  "com.outr"         %% "scribe"    % "2.7.3",
+  "com.typesafe"     %  "config"    % "1.3.4",
+  "org.scalatest"    %% "scalatest" % "3.0.5"      % Test
+)
+
+val datasetComparisonDependencies = List(
+  "org.apache.spark" %% "spark-core" % "2.4.4" % Provided,
```

Comment:
`baseDependencies` is aligned so that `%` symbols are at the same level.  All other lists of dependencies are not aligned. I like the aligned version better.

Reply:
Aligned. Anyway, I am going to move the sbt a little bit more, to make it more like cobrix